doc/man3/SSL_CTX_set_tlsext_ticket_key_cb.pod

```diff
@@ -188,6 +224,10 @@ L<SSL_CTX_sess_number(3)>,
 L<SSL_CTX_sess_set_get_cb(3)>,
 L<SSL_CTX_set_session_id_context(3)>,
 
+=head1 HISTORY
+
+The SSL_CTX_set_tlsext_ticket_key_cb() function was deprecated in OpenSSL 3.0.
```

Comment:
We should mention that `SSL_CTX_set_tlsext_ticket_key_evp_cb` was added in OpenSSL 3.0.